client/src/components/Settings.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import $ from 'jquery';
+
+window.$ = $;
+window.jQuery = $;
```

Comment:
I see you used window alot. set window as global in your webpack config so you don't have to define them in your code